secrets/onepassword/backend.go

```diff
@@ -51,19 +48,15 @@ func (b *Backend) Init(parameters map[string]string) error {
 func (b *Backend) Get(key string) (string, error) {
 	fmt.Println("Retrieving 1password item '" + key + "'.")
 
-	item := b.Client.Get(key)
-	if item == "" {
-		return "", fmt.Errorf("Could not retrieve 1password item '" + key + "'.")
+	value, err := b.Client.Get(b.Vault, key)
+	if value == "" {
+		return "", fmt.Errorf("could not retrieve 1password item '" + key + "'.")
```

Comment:
I think is better to use formatting instead of string concatenation:
```golang
fmt.Errorf("could not retrieve 1password item '%s'",  key)
```

Reply:
Also, I am not sure this should be treated as an error. It could be that the secret is rightfully empty, no? It makes more sense to do the check for `nil` first.

What do you think?

Reply:
You're right. If there is no error it should return the value.

Reply:
Fixed in https://github.com/ContainerSolutions/externalsecret-operator/commit/d0f64360e79ff1f7051b37a85cf8514f7282bb19